dev/deps/spark-deps-hadoop-2.6

```diff
@@ -153,8 +153,8 @@ objenesis/2.5.1/objenesis-2.5.1.jar
 okhttp/3.12.0/okhttp-3.12.0.jar
 okio/1.15.0/okio-1.15.0.jar
 opencsv/2.3/opencsv-2.3.jar
-orc-core/1.5.5/orc-core-1.5.5-nohive.jar
-orc-mapreduce/1.5.5/orc-mapreduce-1.5.5-nohive.jar
+orc-core/1.5.5/nohive/orc-core-1.5.5-nohive.jar
+orc-mapreduce/1.5.5/nohive/orc-mapreduce-1.5.5-nohive.jar
```

Comment:
This is good and can we have four field always for the other jars which doesn't have classifiers? Something like the following?
```
- orc-shims/1.5.5/orc-shims-1.5.5.jar
+ orc-shims/1.5.5//orc-shims-1.5.5.jar
```

Reply:
That's a good idea. Updated to the format suggested.